extensions/wikia/ArticleComments/ArticleCommentsHooks.class.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Wikia\PageHeader\Button;
+
+class ArticleCommentsHooks {
+	public static function onAfterPageHeaderButtons( &$buttons ) {
+		$title = RequestContext::getMain()->getTitle();
+		$service = PageStatsService::newFromTitle( $title );
+		$comments = $service->getCommentsCount();
+
+		if ( $comments && ArticleCommentInit::ArticleCommentCheckTitle( $title ) ) {
+			$label =
+				wfMessage( 'article-comments-comments' )
+					->params( CommentsLikesController::formatCount( $comments ) )
```

Comment:
we should use numParams here: https://github.com/Wikia/app/blob/b8695304136bdd7ae5d1551228c1b75229ec3a0b/includes/Message.php#L278-L293